[ADD] sign: add a 1st doc for Sign on the validity of e-signatures (#805)
odoo/documentation-user: sign/overview/signature_validity.rst

=================================
Validity of electronic signatures
=================================

The legal validity of electronic signatures generated by Odoo depends on the legislation of your
country. Companies doing business abroad should consider electronic signature laws of other
countries as well.

In the European Union
=====================

> The `eIDAS regulation <http://data.europa.eu/eli/reg/2014/910/oj>`_ establishes the framework for
> electronic signatures in all `27 member states of the European Union
> <https://europa.eu/european-union/about-eu/countries_en>`_

Reviewer: ```suggestion
The `eIDAS regulation <http://data.europa.eu/eli/reg/2014/910/oj>`_ establishes the framework for
electronic signatures in all `27 member states of the European Union
<https://europa.eu/european-union/about-eu/countries_en>`_.
```

It distinguishes three types of electronic signatures:

#. Electronic signatures
#. Advanced electronic signatures
#. Qualified electronic signatures

Odoo generates the first type, regular electronic signatures, and these signatures can produce legal
effects in the EU, as the regulation states that “an electronic signature shall not be denied legal
effect and admissibility as evidence in legal proceedings solely on the grounds that it is in an
electronic form or that it does not meet the requirements for qualified electronic signatures.”

Note that electronic signatures may not be automatically recognized as valid. You may need to bring
supporting evidence of a signature’s validity.

In the United States of America
===============================

> The `ESIGN Act (Electronic Signatures in Global and National Commerce Act)
> <https://www.fdic.gov/regulations/compliance/manual/10/X-3.1.pdf>`_, at the interstate and
> international levels, and `UETA (Uniform Electronic Transactions Act)
> <https://www.uniformlaws.org/committees/community-home/librarydocuments?communitykey=2c04b76c-2b7d-4399-977e-d5876ba7e034&tab=librarydocuments>`_,
> at the state level, provide the legal framework for electronic signatures. Note that `Illinois
> <https://www.ilga.gov/legislation/ilcs/ilcs5.asp?ActID=89&>`_ and `New York
> <https://its.ny.gov/electronic-signatures-and-records-act-esra>`_ have not adopted UETA, but similar
> acts instead.

Reviewer: ```suggestion
The `ESIGN Act (Electronic Signatures in Global and National Commerce Act)
<https://www.fdic.gov/regulations/compliance/manual/10/X-3.1.pdf>`_, at the interstate and
international levels, and the `UETA (Uniform Electronic Transactions Act)
<https://www.uniformlaws.org/committees/community-home/librarydocuments?communitykey=2c04b76c-2b7d-4399-977e-d5876ba7e034&tab=librarydocuments>`_,
at the state level, provide the legal framework for electronic signatures. Note that `Illinois
<https://www.ilga.gov/legislation/ilcs/ilcs5.asp?ActID=89&>`_ and `New York
<https://its.ny.gov/electronic-signatures-and-records-act-esra>`_ have not adopted the UETA, but similar
acts instead.
```

Overall, to be recognized as valid, electronic signatures have to meet five criteria:

#. A signer must show a clear intent to sign. For example, using a mouse to draw a signature can
   show intent. The signer must also have the option to opt-out of electronically signing a
   document.
#. A signer must first express or imply their consent to conduct business electronically.
#. The signature must be clearly attributed. In Odoo, metadata, such as the signer’s IP address, is
   added to the signature, which can be used as supporting evidence.

> #. The signature must be associated with the document being signed, for example, by keeping a record
>    that details how the signature was captured.

Reviewer: ```suggestion
#. The signature must be associated with the document being signed, for example, by keeping a record
   detailing how the signature was captured.
```

> #. Electronically signed documents need to be retained and available for later reference by all
>    parties involved, for example by providing the signer either a fully-executed copy or the option
>    to download a copy.

Reviewer: ```suggestion
#. Electronically signed documents need to be retained and available for later reference by all
   parties involved, for example, by providing the signer either a fully-executed copy or the option
   to download a copy.
```

.. note::
   The information provided here does not constitute legal advice; it is provided for general
   informational purposes only. As laws governing electronic signatures evolve rapidly, we cannot

>    guarantee that the information is current. We advise you to should contact a local attorney to

Reviewer: ```suggestion
   guarantee that the information is up to date. We advise you to should contact a local attorney to
```
